lib/msal-common/src/config/ClientConfiguration.ts

```diff
@@ -53,6 +54,7 @@ export type CommonClientConfiguration = {
     authOptions: Required<AuthOptions>;
     systemOptions: Required<SystemOptions>;
     loggerOptions: Required<LoggerOptions>;
+    cacheOptions: CacheOptions;
```

Comment:
Required<CacheOptions>?

Reply:
Updated. What's our criteria for which types we pull with Required and which we don't?

Reply:
Customer facing = not `Required`, during initialization we assign defaults to everything so any internal references should be `Required`. This helps us by not needing to constantly check whether things are defined or not